engine/execution/migration/badgerpebble.go

```diff
@@ -1,201 +0,0 @@
-package migration
```

Comment:
This file was used when migrating part of EN's protocol data, but in the end, we decided to migrate all the protocol data using the `db-migration` util. 

This file is deprecated, and removed now.